integration/web-e2e_test.go

```diff
@@ -0,0 +1,44 @@
+package integration
```

Comment:
As this test requires the `webassets_embed` tag down the line, you might want to put the tag in this file as well. Else it might run anf fail when we do `go test` or in go's integration tests.